plugin-outbound-whatsapp/src/utils/templates.js

```diff
@@ -1,4 +1,3 @@
 export const templates = [
-  'Thank you for purchasing Saul Goodman! We value your feedback and would like to learn more about your experience.',
-  'Hi Saul Goodman, were we able to solve the issue that you were facing?'
+  'Hi customer, were we able to solve the issue that you were facing?'
```

Comment:
No need to change this for now, we will remove this file soon to add a field to upload templates